lib/pipedrive.rb

```diff
@@ -7,8 +7,8 @@
 require 'active_support/inflector'
 
 require 'hashie'
-require 'faraday'
 require 'faraday_middleware'
+require 'faraday'
```

Comment:
Chang the loading order to prevent error
```
Gem::ConflictError:
  Unable to activate faraday_middleware-1.2.0, because faraday-2.7.9 conflicts with faraday (~> 1.0)

```